tests/k8s/test_deployer.py

```diff
@@ -1,116 +1,191 @@
 #!/usr/bin/env python
 # -*- coding: utf-8
 
-from pprint import pformat
-
 import pytest
+import mock
+
+from k8s.client import NotFound
 from k8s.models.common import ObjectMeta
-from k8s.models.deployment import Deployment, DeploymentSpec, LabelSelector
+from k8s.models.deployment import Deployment, DeploymentSpec, LabelSelector, RollingUpdateDeployment, \
+    DeploymentStrategy
 from k8s.models.pod import ContainerPort, Container, LocalObjectReference, Probe, HTTPGetAction, TCPSocketAction, \
     PodTemplateSpec, PodSpec
-from util import get_vcr
-
-vcr = get_vcr(__file__)
 
 NAME = "my-name"
 NAMESPACE = "my-namespace"
 
 
-@pytest.mark.usefixtures("logger", "k8s_config")
+@pytest.mark.usefixtures("k8s_config")
 class TestDeployer(object):
+    @pytest.fixture
```

Comment:
I think the next test we convert away from vcr, we should move these fixtures to `conftest.py` and share them across all tests.

Reply:
Sounds like a good plan!